### PR TITLE
Update docker information with extension info

### PR DESCRIPTION
**Pull request by JTeijema**
This PR will make some changes to the docker documentation in the GitHub repository. It will explain how to add your custom packages to your images.

### CODE REVIEW

**Comment by JTeijema** [comment]
Read the updated [readme](https://github.com/JTeijema/asreview/blob/extent-docker-info/docker/README.md)

File: docker/README.md [hunk 1]
```diff
@@ -88,8 +88,31 @@ directory as the dockerfile is in:
 docker build -t asreview:1.0rc0 .
 ```
 
+![instruction image](./img/dockerfile.png)
```
Comment (J535D165) [inline docker/README.md:91]: I think the added value of this image is low.

File: docker/README.md [hunk 2]
```diff
@@ -88,8 +88,31 @@ directory as the dockerfile is in:
...
+
+### Add you own extension to your image
+
+It's possible to test your own packages by adding them to your image. Next to
+the `Dockerfile` file, create a folder with a functional package inside. Then,
+add the following code to the file;
+
+```dockerfile
+FROM python:3.8
+RUN pip install asreview==1.0rc0
```
Comment (J535D165) [inline docker/README.md:107]:
```suggestion
RUN pip install asreview
```

**Review by J535D165 — CHANGES_REQUESTED** [review]
Very welcome. This makes the life of a (data) scientist a lot easier.

File: docker/README.md (hunk 1 above)
Comment (JTeijema) [reply docker/README.md]: I agree 👍🏻

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Merci, nice addition.

**Pull request merged**